projects/frontend/data-pipelines/gui/package.json

```diff
@@ -16,7 +16,7 @@
     "test": "npm run ng -- test --watch=false",
     "test:headless": "npm run test -- --browsers=ChromeHeadless_No_Sandbox",
     "lint": "npm run ng -- lint",
-    "e2e": "npx cypress run --browser chrome --env grepTags=${FRONTEND_E2E_CYPRESS_GREP_TAGS},OAUTH2_API_TOKEN=${VDK_API_TOKEN},CYPRESS_TERMINAL_LOGS='./e2e/logs'",
+    "e2e": "([ ! -z ${OAUTH2_API_TOKEN} ] || echo 'WARN: No OAUTH2_API_TOKEN set.') && npx cypress run --browser chrome --env grepTags=${FRONTEND_E2E_CYPRESS_GREP_TAGS},OAUTH2_API_TOKEN=${OAUTH2_API_TOKEN},CYPRESS_TERMINAL_LOGS='./e2e/logs' --headed",
```

Comment:
What are the implications of running in headed mode? I know it works around that bug we had with headless mode, but what's the trade-off here? Does it consume more resources?

Reply:
That's a good question, thanks for bringing it up;
I added the answer to it, to the issue opened,
and listed that in the PR description.

Reply:
Let's keep an eye on it for now. If there are problems with running out of resources, we can get back to it.